docs/content/en/docs/reference/metrics.md

```diff
@@ -0,0 +1,422 @@
+---
+title: "Metrics Reference"
```

Comment:
I would remove this as it's already under the Reference section, like for the other ones.
```suggestion
title: "Metrics"
```